waterfall/java/com/google/waterfall/usb/BUILD

```diff
@@ -0,0 +1,103 @@
+exports_files(
+    ["AndroidManifest.xml"],
+)
+
+filegroup(
+    name = "resource_files",
+    srcs = glob(["res/**"]),
+)
+
+# TODO(mauriciogg): this needs to be proguarded because of dagger
+android_binary(
+    name = "usb_service",
+    manifest = "AndroidManifest.xml",
+    resource_files = [":resource_files"],
+    deps = [
+        ":usb_service_lib",
+    ],
+)
+
+android_library(
+    name = "usb_service_lib",
+    srcs = [
+        "AndroidUsbModule.java",
+        "UsbService.java",
+    ],
+    manifest = "AndroidManifest.xml",
+    resource_files = [":resource_files"],
+    deps = [
+        ":annotations",
+        ":mock_usb_accessory",
```

Comment:
why are mocks part of the main android_library ?

Reply:
Removed mock dependencies. They were unused.